include/dlpack/dlpack.h

```diff
@@ -126,6 +126,11 @@ typedef enum {
    * (C/C++/Python layout: compact struct per complex number)
    */
   kDLComplex = 5U,
+  /*!
+   * \brief boolean
+   * The underlying storage size of bool is 8 bits, as per common array library convention
```

Comment:
it is actually better to annotate in another location(where i commented). Since here it can cause confusion that Bool always is 8bits regardless of the bit field. But in reality the current convention can support things like Bool(bits=1, lanes=32) for 32 bit bitmask

Reply:
I would think the examples are meant to be canonical, so for example we would not allow alternative interpretations for common types like `float4` (and `bool`), otherwise it'd destroy the purpose of having an agreed upon format to facilitate exchange?

Reply:
That is fair, the main concern i had about the current one is that a reader may expect `Bool(bits=1, lane=1)` to have 8 bits. Instead, i would write we expect bool to be `Bool(bits=8, lanes=1)` as a canonical way of storage.

I am not stating that `Bool(bits=1, lanes=32)` should be supported right away, just avoid confusion and leave doors for future compatibility

Reply:
I see, thanks for clarification. Updated.